src/packages/core/components/input-color/input-color.element.ts

```diff
@@ -11,6 +11,15 @@ import type { UUIColorSwatchesEvent } from '@umbraco-cms/backoffice/external/uui
  */
 @customElement('umb-input-color')
 export class UmbInputColorElement extends UUIFormControlMixin(UmbLitElement, '') {
+	/**
+	 * Sets the input to readonly mode, meaning value cannot be changed but still able to read and select its content.
+	 * @type {boolean}
+	 * @attr
+	 * @default false
+	 */
+	@property({ type: Boolean, reflect: true })
+	readonly = false;
+
```

Comment:
Note it also require to set `readonly` attribute on the underlying rendered `<uui-color-swatches>` element, which hasn't be implemented yet: https://github.com/umbraco/Umbraco.UI/pull/932